src/main/webapp/app/shared/link-preview/components/link-preview/link-preview.component.scss

```diff
@@ -0,0 +1,85 @@
+.loading-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100px;
+}
+
+.loading-spinner {
+    display: flex;
+    align-items: center;
+}
+
+.spinner {
+    border: 4px solid #f3f3f3; /* Light gray */
+    border-radius: 50%;
+    width: 40px;
+    height: 40px;
+    animation: spin 1s linear infinite;
+}
+
+@keyframes spin {
+    0% {
+        transform: rotate(0deg);
+    }
+    100% {
+        transform: rotate(360deg);
+    }
+}
+
+.loading-container,
+.preview-content {
+    margin-left: 10px; /* Adjust the margin as needed */
```

Comment:
What does this comment say? Should someone adjust this later on?

Reply:
No, I left the comment when i was fixing some styles and forgot to remove.